scripts/sftp_download.py

```diff
@@ -0,0 +1,33 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2014-2022 ftrack
+
+import ftrack_api.structure.id
+import ftrack_api.accessor.disk
+import os
+
+from sftp import SFTPAccessor
+
+"""
+Copies a component from a sftp location to a local location
+"""
+session = ftrack_api.Session()
+sftp_location = session.query('Location where name is "studio.sftp"').one()
```

Comment:
@iwootten 

might worth replacing it with something like

`location = session.ensure('Location', {"name":"studio.sftp"})`